examples/tox21/predictor.py

```diff
@@ -50,23 +52,29 @@ def build_predictor(method, n_unit, conv_layers, class_num):
 class GraphConvPredictor(chainer.Chain):
     """Wrapper class that combines a graph convolution and MLP."""
 
-    def __init__(self, graph_conv, mlp):
+    def __init__(self, graph_conv, mlp=None):
         """Constructor
 
         Args:
             graph_conv: graph convolution network to obtain molecule feature
                         representation
-            mlp: multi layer perceptron, used as final connected layer
+            mlp: multi layer perceptron, used as final connected layer.
+                It can be `None` if no operation is necessary after
+                `graph_conv` calculation.
         """
 
         super(GraphConvPredictor, self).__init__()
         with self.init_scope():
             self.graph_conv = graph_conv
+            if isinstance(mlp, chainer.Link):
```

Comment:
Is this check necessary?

Reply:
Oh sorry. I would like to confirm. If mlp == `None`,  will it raise Error in `with init_scope` statement?

Reply:
If mlp is not Link (I'm assuming it can be None or even function), then I thought it is better to set this value outside of `init_scope` but I don't know.